pkg/apiserver/middlewares/v1/jwt.go

```diff
@@ -43,39 +47,103 @@ func IdentityHandler(c *gin.Context) interface{} {
 	}
 }
 
+//var AllowedOU = "bouncer"
+
+func (j *JWT) SetAllowedOUs(allowedOu []string) error {
+	for _, ou := range allowedOu {
+		//just drop empty strings, I guess ?
+		if ou == "" {
+			log.Warningf("Ignoring empty OU string in Agents Allowed OU (jwt)")
+			continue
+		}
+		j.AllowedOu = append(j.AllowedOu, ou)
+	}
+	log.Infof("%p Allowed Agents OU : %v", j, j.AllowedOu)
+	return nil
+}
+
 func (j *JWT) Authenticator(c *gin.Context) (interface{}, error) {
```

Comment:
we need to honor the AuthType flag of the agent (do not allow a agent w/ AuthType set to "password" to present CERT and vice-versa)